### PR TITLE
chore: spfx readme

### PR DESCRIPTION
**Pull request by yuqizhou77**

### CODE REVIEW

File: templates/scenarios/ts/spfx-tab/src/README.md [hunk 1]
```diff
@@ -22,18 +22,14 @@ Learn more about the changes in this pre-release at [https://aka.ms/teamsfx-v5.0
...
-
 ## Applies to
 
 - [SharePoint Framework](https://aka.ms/spfx)
 - [Microsoft 365 tenant](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-developer-tenant)
 
 ## Prerequisites
 
-> - [Node.js](https://nodejs.org/en/), supported versions: 16
+> - [SharePoint Framework development environment](https://aka.ms/teamsfx-spfx-dev-environment-setup)
```
Comment (summzhan) [inline templates/scenarios/ts/spfx-tab/src/README.md:32]: Set up SharePoint Framework development environment.

File: templates/scenarios/ts/spfx-tab/src/README.md [hunk 2]
```diff
@@ -22,18 +22,14 @@ Learn more about the changes in this pre-release at [https://aka.ms/teamsfx-v5.0
...
 > - An Microsoft 365 account. Get your own free Microsoft 365 tenant from [Microsoft 365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
```
Comment (summzhan) [inline templates/scenarios/ts/spfx-tab/src/README.md:33]: Download Visual Studio Code and install [Teams Toolkit for Visual Studio Code Extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).

**Review by summzhan — CHANGES_REQUESTED** [review]
please resolve comments

File: templates/scenarios/ts/spfx-tab/src/README.md (hunk 2 above)
Comment (yuqizhou77) [reply templates/scenarios/ts/spfx-tab/src/README.md:33]: Download [Visual Studio Code](https://code.visualstudio.com) and install [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) or download TeamsFx CLI. - how about this one?

Comment (summzhan) [reply templates/scenarios/ts/spfx-tab/src/README.md:33]: ok

**Review by summzhan — APPROVED** [review]

**Pull request merged**